modules/dom_methods.js

```diff
@@ -0,0 +1,45 @@
+// eslint-disable-next-line import/no-mutable-exports
+let books = [];
+
```

Comment:
- [ ] Great job, please don't disable linters, it better to solve the issues that come up rather than disabling them 👍.

Reply:
@Wes-Isaac , I had trouble overcoming that